packages/mui-base/src/TextareaAutosize/TextareaAutosize.tsx

```diff
@@ -209,12 +253,12 @@ const TextareaAutosize = React.forwardRef(function TextareaAutosize(props, ref)
         onChange={handleChange}
         ref={handleRef}
         // Apply the rows prop to get a "correct" first SSR paint
-        rows={minRows}
+        rows={Number(minRows)}
```

Comment:
If it was working before, and there is no bug, better not add bundle size to make types happy.

```suggestion
        rows={minRows as any}
```